cozy/wf.py

```diff
@@ -47,8 +47,8 @@ def exp_wf_nonrecursive(solver, e : Exp, context : Context, pool = RUNTIME_POOL,
         raise ExpIsNotWf(e, e, "EDrop* in state position")
     if isinstance(e, EFlatMap) and not at_runtime:
         raise ExpIsNotWf(e, e, "EFlatMap in state position")
-    if not at_runtime and isinstance(e, EBinOp) and e.type == INT:
-        raise ExpIsNotWf(e, e, "integer arithmetic in state position")
+#    if not at_runtime and isinstance(e, EBinOp) and e.type == INT:
+#        raise ExpIsNotWf(e, e, "integer arithmetic in state position")
```

Comment:
Remove these lines (or guard them with an `Option` that is false by default) instead of commenting them out.